math/mathmod/Portfile

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                mathmod
+
+categories          math graphics
+license             GPL-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+homepage            https://www.facebook.com/parisolab
+
+description         Mathematical modelling software
```

Comment:
In US English we always write "modeling".

Reply:
Yeah, but both spellings are correct.